server/views/partials/recommendation/recallTypeFTR.njk

```diff
@@ -4,12 +4,51 @@
     For sentences under 12 months, recalls must be fixed term unless either:
     <ul>
         <li>the person has been charged with a serious offence (murder or an offence listed in
-            <a target="_blank" rel="noopener noreferrer" href="https://www.legislation.gov.uk/ukpga/2020/17/schedule/18">Schedule 18 of the Sentencing Act 2020</a>)
+            <a target="_blank" rel="noopener noreferrer" href="https://www.legislation.gov.uk/ukpga/2020/17/schedule/18">Schedule 18
+                of the Sentencing Act 2020</a>)
         </li>
         <li>the MAPPA level is 2 or 3 at the point of recall</li>
     </ul>
     This applies to people aged 18 and over only.
 </p>
+<p>
+    You have said that:
+    <ul>
+        <li>{{ recommendation.personOnProbation.name }}
+            <strong>is</strong>
+            {% if recommendation.isUnder18 == false or recommendation.isUnder18 == 'NO' %}
+                <strong>not</strong>
+            {% endif %}
+            under 18</li>
+        <li>the sentence
+            <strong>is</strong>
+            {% if recommendation.isSentence12MonthsOrOver == false or recommendation.isSentence12MonthsOrOver == 'NO' %}
+                <strong>not</strong>
+            {% endif %}
+            12 months or over</li>
+        <li>the MAPPA level
+            <strong>is</strong>
+            {% if recommendation.isMappaLevelAbove1 == false or recommendation.isMappaLevelAbove1 == 'NO' %}
+                <strong>not</strong>
+            {% endif %}
+            above 1</li>
+        <li>{{ recommendation.personOnProbation.name }}
+            <strong>has</strong>
+            {% if recommendation.hasBeenConvictedOfSeriousOffence == false or recommendation.hasBeenConvictedOfSeriousOffence == 'NO' %}
+                <strong>not</strong>
+            {% endif %}
+            been charged with a serious offence.</li>
+    </ul>
+    {% if recommendation.isUnder18 or recommendation.isSentence12MonthsOrOver or isMappaLevelAbove1 or hasBeenConvictedOfSeriousOffence %}
```

Comment:
Mismatch here with `recommendation.` not being included on the last 2 conditions.

There is an assumption here that these values are boolean, whereas the code above is checking for 'NO'

Reply:
corrected